docs/zh-CN/sql-reference/sql-statements/Data Manipulation/ROUTINE LOAD.md

```diff
@@ -137,15 +137,23 @@ under the License.
             采样窗口内，允许的最大错误行数。必须大于等于0。默认是 0，即不允许有错误行。
             采样窗口为 max_batch_rows * 10。即如果在采样窗口内，错误行数大于 max_error_number，则会导致例行作业被暂停，需要人工介入检查数据质量问题。
             被 where 条件过滤掉的行不算错误行。
-        
+
         4. strict_mode
 
             是否开启严格模式，默认为开启。如果开启后，非空原始数据的列类型变换如果结果为 NULL，则会被过滤。指定方式为 "strict_mode" = "true"
 
         5. timezone
-            
+
             指定导入作业所使用的时区。默认为使用 Session 的 timezone 参数。该参数会影响所有导入涉及的和时区有关的函数结果。
 
+        6. format
+
+            指定导入数据格式，默认是csv，支持json格式。
+
+        7. jsonpath、jsonpath_file
```

Comment:
1. jsonpaths and jsonpaths_file seems a better name
2. should give explain about format of jsonpaths
3. should give explain about how this works